src/card.css

```diff
@@ -0,0 +1,90 @@
+* {
+  box-sizing: border-box;
+}
+
+.card-box {
+  width: 200px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+
+  font-family: "Roboto", Helvetica, sans-serif;
+  color: #060b35;
+  font-weight: 400;
+  font-size: 10px;
+  border-radius: 5px;
+}
+
+.card__img {
+  width: 160px;
+}
+
+.card__img-imac {
+  width: 100%;
+  height: 134px;
+  margin: 32px 19px 37px;
+}
+
+.card__information,
+.card__price,
+.card__rating,
+.card__button {
+  margin: 0 16px 16px;
+}
+
+.card__rating,
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__rating {
+  margin-bottom: 24px;
+}
+
+.card__information-name {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 150%;
+}
+
+.card__information-code {
+  margin-top: 4px;
+  color: #616070;
+}
+
+.card__price-hint {
+  font-weight: 400;
+  font-size: 12px;
+  color: #616070;
+}
+
+.card__price-number {
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.card__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 10.5px;
```

Comment:
wtf

Reply:
oh, maybe it`s cos box-sizing: border-box, but 0.5 px sounds insecure, but mayabe ok